### PR TITLE
Edit email template endpoint

### PR DESCRIPTION
**Pull request by djino97**
- Created new endpoint POST /editemailtemplate
- Added new command (EditEmailTemplateCommand), methods repositories and validator
- Added check rights user

Tests not added

### CODE REVIEW

**Comment by lalagvana** [comment]
Add RabbitMQ configuration in appsettings.json (take it from PR Send email logic)

**Comment by djino97** [comment]
> Add RabbitMQ configuration in appsettings.json (take it from PR Send email logic)

Added

File: src/MessageService.Mappers/EmailTemplateMapper.cs [hunk 1]
```diff
@@ -24,5 +25,23 @@ public DbEmailTemplate Map(EmailTemplate emailTemplate)
...
+        public DbEmailTemplate Map(EditEmailTemplateRequest emailTemplate, DbEmailTemplate dbEmailTemplate)
+        {
+            if (emailTemplate == null)
+            {
+                throw new ArgumentNullException(nameof(emailTemplate));
+            }
+
+            return new DbEmailTemplate
+            {
+                Id = Guid.NewGuid(),
```
Comment (lalagvana) [inline src/MessageService.Mappers/EmailTemplateMapper.cs:38]: There should be Id from EditEmailTemplateRequest, not new Guid, I suppose.

File: src/MessageService.Validation/EditEmailTemplateValidator.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+            RuleFor(et => et.Subject)
+                .NotEmpty()
+                .MaximumLength(120)
+                .WithMessage("Subject too long");
+
+            RuleFor(et => et.Body)
+                .NotEmpty()
+                .MaximumLength(1000)
+                .WithMessage("Subject too long");
```
Comment (lalagvana) [inline src/MessageService.Validation/EditEmailTemplateValidator.cs:22]: Why it can't be as long as the author wants?

Comment (djino97) [reply src/MessageService.Validation/EditEmailTemplateValidator.cs]: Fixed

**Review by lalagvana — APPROVED** [review]

**Review by DarkFelas — APPROVED** [review]

**Pull request merged**